### PR TITLE
[DX-1498] double alias /getting-started/installation/with-tyk-on-premises/

### PR DESCRIPTION
**Pull request by letzya**
### For internal users - Please add a Jira DX PR ticket to the subject!
<br>

---

### Preview Link
https://deploy-preview-5031--tyk-docs.netlify.app/docs/nightly//getting-started/installation/with-tyk-on-premises/

[install preview](https://deploy-preview-5031--tyk-docs.netlify.app/docs/nightly/tyk-self-managed/install)

### Description

<br>

#### Screenshots (if appropriate)
<br>

## Checklist

- [x] I have added a **preview link** to the PR description.
- [x] I have [reviewed the guidelines](https://github.com/TykTechnologies/tyk-docs/blob/master/CONTRIBUTING.md) for contributing to this repository.
- [x] I have [read the technical guidelines](https://github.com/TykTechnologies/tyk-docs/blob/master/CONTRIBUTING-TECHNICAL-GUIDE.md) for contributing to this repository.
- [x] Make sure you have started *your change* off *our latest `master`*.
- [x] I **labelled** the PR

___

### **PR Type**
Documentation

___

### **Description**
- Improved formatting for buttons in the self-managed licensing section.
- Removed duplicate alias in the Tyk On-Premises documentation.
- Corrected alias format and improved introductory text in the Tyk Self-Managed installation guide.

___

### **Changes walkthrough** 📝
<table><thead><tr><th></th><th align="left">Relevant files</th></tr></thead><tbody><tr><td><strong>Documentation</strong></td><td><table>
<tr>
  <td>
    <details>
      <summary><strong>self-managed-licensing-include.md</strong><dd><code>Improved formatting for licensing section buttons</code>&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; </dd></summary>
<hr>

tyk-docs/content/shared/self-managed-licensing-include.md

<li>Added a newline for better formatting before the "Free trial" button.<br> <li> Adjusted the formatting of the "Contact us" button for consistency.<br>

</details>

  </td>
  <td><a href="https://github.com/TykTechnologies/tyk-docs/pull/5031/files#diff-b46c343a3cf502455a2e6f2507461b16ef81846d3ef643d79786035d023f46e1">+4/-2</a>&nbsp; &nbsp; &nbsp; </td>

</tr>                    

<tr>
  <td>
    <details>
      <summary><strong>tyk-on-premises.md</strong><dd><code>Removed duplicate alias in Tyk On-Premises documentation</code>&nbsp; </dd></summary>
<hr>

tyk-docs/content/tyk-on-premises.md

<li>Removed duplicate alias for <br><code>/getting-started/installation/with-tyk-on-premises/</code>.<br>

</details>

  </td>
  <td><a href="https://github.com/TykTechnologies/tyk-docs/pull/5031/files#diff-b0f038f8e9334fff1d674de39dfcc2eb93738df5edc0cfea5d4ac4c3205b7808">+0/-1</a>&nbsp; &nbsp; &nbsp; </td>

</tr>                    

<tr>
  <td>
    <details>
      <summary><strong>install.md</strong><dd><code>Corrected aliases and improved introductory text</code>&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; </dd></summary>
<hr>

tyk-docs/content/tyk-self-managed/install.md

<li>Corrected alias format for consistency.<br> <li> Improved the introductory text for trial license and installation <br>options.<br>

</details>

  </td>
  <td><a href="https://github.com/TykTechnologies/tyk-docs/pull/5031/files#diff-79e3085f7b898531e3e8990926b3d7dd929ec10994721efe971b634dc26e0e14">+3/-4</a>&nbsp; &nbsp; &nbsp; </td>

</tr>                    
</table></td></tr></tr></tbody></table>

___

> 💡 **PR-Agent usage**:
>Comment `/help` on the PR to get a list of all available PR-Agent tools and their descriptions

### CODE REVIEW

**Comment by letzya** [comment]
@Keithwachira fyi - we need a gh action for it

File: tyk-docs/content/tyk-self-managed/install.md [hunk 1]
```diff
@@ -10,16 +10,15 @@ menu:
...
   - /tyk-self-managed/istio/
-  - "tyk-self-managed/install"
+  - /tyk-self-managed/install/
   - /getting-started/installation/with-tyk-on-premises/
   - /get-started/with-tyk-on-premise/installation/
 ---
 
-First, click here to get a free trial license - 
-</br>
+First, click here to get a trial license
```
Comment (dcs3spp) [inline tyk-docs/content/tyk-self-managed/install.md:18]:
```suggestion
First, click here to get a trial license

</br>

```

**Review by dcs3spp — APPROVED** [review]
PR LGTM and approved with minor suggestion for adding button below click here sentence

**Pull request merged**